datastore/importers/nmdb-import-ipconfig/Parser.cpp

```diff
@@ -63,14 +73,14 @@ Parser::Parser() : Parser::base_type(start)
      | ("Connection-specific DNS Suffix" >> dots > -token
           [(pnx::bind(&Parser::setIfaceDnsSuffix, this, qi::_1))] > qi::eol)
      | ("Default Gateway" >> dots > *(getIp
-          [(pnx::bind(&Parser::addRoute, this, qi::_1))] > qi::eol) > -qi::eol)
+          [(pnx::bind(&Parser::addRoute, this, qi::_1))] > qi::eol))
```

Comment:
This probably needs to be something like:
```
| ("Default Gateway" >> dots > (+(getIp [(pnx::bind(&Parser::addRoute, this, qi::_1))] > qi::eol) | qi::eol)
```
Primarily because "Default Gateway" can have either 1) one or more IPs or 2) nothing after it (e.g., https://social.technet.microsoft.com/Forums/getfile/828925) and the current logic (in this revision) fails to account for the 2nd case now.

Note: The previous logic would still be wrong given it can be too greedy when "Default Gateway" is the last configuration setting since it will consume two newlines instead of the intended one.